lib/backup/logger/logfile.rb

```diff
@@ -102,8 +107,8 @@ def setup_logfile
           path = File.join(Backup::Config.root_path, path)
         end
         FileUtils.mkdir_p(path)
-        log_file = @options.log_file || 'backup.log'
-        path = File.join(path, log_file)
+        log_filename = @options.log_filename.strip.empty? ? 'backup.log' : @options.log_filename
```

Comment:
is `.strip` necessary here? The spaces are already stripped from the string when it's set in the `log_filename` method